admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MapperTest.java

```diff
@@ -137,6 +140,52 @@ public void testSetCreateMetaDataWithIsActive() {
 		assertNotNull(entity.getCreatedBy());
 		assertFalse(entity.getIsActive());
 	}
+	@Test
+	public void mapUpdateHolidaysTest001(){
+		List<Holiday> holidays=new ArrayList<>();
+		Holiday holiday=new Holiday();
+		holiday.setHolidayDate(LocalDate.now());
+		holiday.setHolidayId(123);
+		holiday.setLangCode("eng");
+		holiday.setIsActive(true);
+		holidays.add(holiday);
+
+		List<HolidayUpdateDto> holidayDtos = new ArrayList<>();
+		HolidayUpdateDto holidayUpdateDto=new HolidayUpdateDto();
+		holidayUpdateDto.setHolidayName("name");
+		holidayUpdateDto.setHolidayDesc("desc");
+		holidayUpdateDto.setHolidayId(345);
+		holidayUpdateDto.setHolidayDate(LocalDate.now());
+		holidayUpdateDto.setHolidayId(7);
+		holidayUpdateDto.setLangCode("eng");
+		holidayUpdateDto.setLocationCode("location123");
+		holidayDtos.add(holidayUpdateDto);
+
+		MapperUtils.mapUpdateHolidays(holidays);
+	}
+	@Test
+	public void mapRegisteredDeviceDtoTest001(){
+		RegisteredDevicePostReqDto dto=new RegisteredDevicePostReqDto();
+		dto.setDeviceId("device_id");
+		dto.setStatusCode("status_code");
+		dto.setDeviceSubId("device_subid");
+		dto.setPurpose("purpose");
+		dto.setFirmware("firmware");
+		dto.setExpiryDate(LocalDateTime.now());
+		dto.setCertificationLevel("certification_level");
+		dto.setFoundationalTPId("functional_tpid");
+		DigitalIdDeviceRegisterDto digitalIdDto=new DigitalIdDeviceRegisterDto();
+		digitalIdDto.setDp("dp");
+		digitalIdDto.setMake("make");
+		digitalIdDto.setModel("model");
+		digitalIdDto.setDpId("dp_id");
+		digitalIdDto.setDateTime("date_time");
+		digitalIdDto.setDeviceSTypeCode("device_st_type_code");
+		digitalIdDto.setDeviceTypeCode("device_type_code");
+		digitalIdDto.setSerialNo("serial_no");
+		dto.setDigitalIdDto(digitalIdDto);
+		MapperUtils.mapRegisteredDeviceDto(dto,"digital_id_json");
```

Comment:
The test case must assert either exception or a result.

Reply:
Added an assertions in the test methods and updated the method names